src/status_im/subs/wallet/wallet.cljs

```diff
@@ -464,22 +465,49 @@
  keep-operable-accounts)
 
 (rf/reg-sub
- :wallet/operable-accounts-without-watched-accounts
+ :wallet/operable-accounts
  :<- [:wallet/accounts-without-watched-accounts]
  keep-operable-accounts)
```

Comment:
Renamed because watched accounts can't be  operable accounts, since we don't own them.
The name now is more simple and short.

CC: @smohamedjavid  wdyt?